packages/connect/e2e/test-yarn-install.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# validate that installing connect package using yarn works
+
+set -e
+
+trap "cd .. && rm -rf connect-implementation" EXIT
+
+npm --version
+node --version
+
+mkdir connect-implementation
+cd connect-implementation
+npm init -y
+touch yarn.lock
+yarn add @trezor/connect
+yarn add @trezor/connect-web
```

Comment:
I do not see this file used or documented anywhere. When is it supposed to be used?

Reply:
oh, it should be used in CI. test-npm install.sh is already in utils.yml

Reply:
oh it is there in this PR

Reply:
Ok, true, for some reason I did not find it.